.github/workflows/website.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         uses: actions-rs/toolchain@v1
         with:
           profile: minimal
-          toolchain: nightly-2023-05-24
+          toolchain: nightly-2024-05-14
```

Comment:
I think this might be able to go to `nightly-latest`? Opinions?

Reply:
I always lock rustc to a specific version, so that way it's reproducible and consistent for everybody. If it's `nightly-latest` then it can cause breakage.

Note that the CI matches the version in `rust-toolchain.yaml`, so that way the CI will match local dev builds.

Reply:
Also note that locking the version when using nightly is [standard best practice](https://www.justanotherdot.com/posts/how-to-pick-stable-beta-or-nightly-rust).

Reply:
Agreed